game/menu.cc

```diff
@@ -4,9 +4,7 @@
 #include "fs.hh"
 #include "game.hh"
 
-MenuOption::MenuOption(std::string const& nm, std::string const& comm, MenuImage img):
-  type(), value(), newValue(), callback(), image(img), name(nm), comment(comm), namePtr(), commentPtr()
-{}
+MenuOption::MenuOption(std::string const& nm, std::string const& comm, MenuImage img): type(), value(), newValue(), options(std::make_shared<MenuOptions>()), callback(), image(img), name(nm), comment(comm), namePtr(), commentPtr() {}
```

Comment:
no need to initialize a shared_ptr to null, it is already done in the defautl constructor